vtk-cdat/meta.yaml.in

```diff
@@ -3,7 +3,7 @@ package:
     version: 7.1.0.@VERSION@
 
 source:
-    git_rev: master
+    git_rev: f03521b50c4b630d4dcf23d237ef2d9470581306
```

Comment:
@danlipsa following your recommendation, `prep_build.py` now automatically appends the sha1 to the date for the version number. The only issue is that multiple builds on the same day might lead to the wrong one being detected as `latest` but it's a minor issue.